loader/loader.go

```diff
@@ -0,0 +1,57 @@
+package loader
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+)
+
+type StandData struct {
+	StandID        string  `json:"standId"`
+	Stand         string  `json:"stand"`
+	Power       string  `json:"power"`
+	Speed       string  `json:"speed"`
+	Range       string  `json:"range"`
+	Persistence       string  `json:"persistence"`
+	Precision       string  `json:"precision"`
+	Development       string  `json:"development"`
+
+}
+
+func LoadData(r io.Reader) *[]*StandData {
+	reader := csv.NewReader(r)
+
+	ret := make([]*StandData, 0, 0)
```

Comment:
If you don't know the size of the slice, it would be better to just initialize it like: `ret := []*StanData{}`